### PR TITLE
[wix-storybook-utils] `<Playground/>` - autocomplete

### PR DESCRIPTION
**Pull request by egidijusja**

### CODE REVIEW

File: packages/wix-storybook-utils/src/LiveCodeExample/Editor.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+              const indent = codemirror.getOption('indentUnit');
+              const spaces = Array(indent + 1).join(' ');
+              codemirror.replaceSelection(spaces);
+            }
+          },
+          "'<'": completeAfter,
+          'Cmd-I': completeIfInTag,
+          'Ctrl-I': completeIfInTag,
+          "'='": completeIfInTag,
+          "' '": completeIfInTagNewAttribute,
```
Comment (mykas) [inline packages/wix-storybook-utils/src/LiveCodeExample/Editor.tsx:146]: Lets see if there is a parameter called `Space` instead this

Comment (mykas) [reply packages/wix-storybook-utils/src/LiveCodeExample/Editor.tsx:146]:
```suggestion
          Space:completeIfInTagNewAttribute,
```

maybe?

File: packages/wix-storybook-utils/src/LiveCodeExample/Editor.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        autoCloseTags: true,
+        autoCloseBrackets: true,
+        theme: 'wsr',
+        viewportMargin: 50,
+        lineNumbers: true,
+        hintOptions: {
+          schemaInfo: hints,
+        } as CodeMirror.ShowHintOptions,
+        extraKeys: {
+          Tab: codemirror => {
```
Comment (mykas) [inline packages/wix-storybook-utils/src/LiveCodeExample/Editor.tsx:133]: Lest extract thus function too and call it: `handleIndentation`

File: packages/wix-storybook-utils/src/Playground/index.tsx [hunk 3]
```diff
@@ -103,6 +104,7 @@ const Playground: React.FunctionComponent<Props> = ({
           });
         }}
         previewProps={{ className: styles.overflowPreview }}
+        hints={getHints(rest.scope)}
```
Comment (mykas) [inline packages/wix-storybook-utils/src/Playground/index.tsx:107]: Lets move this logic inside LiveCodeExample instead so every code editor in storybook will benefit

File: packages/wix-storybook-utils/src/LiveCodeExample/Editor.tsx (hunk 1 above)
Comment (egidijusja) [reply packages/wix-storybook-utils/src/LiveCodeExample/Editor.tsx]: Yeah it exists, I changed it to `Space`

**Pull request merged**